src/TodoList.jsx

```diff
@@ -3,35 +3,46 @@ import './App.css'
 
 
 function TodoLIst() {
+  const [todo, setTodo] = useState([]);
+const clickhandler=()=>{
+ //alert(todo);
+}
+const change = event =>{
+  setTodo(event.target.value)
   
+}
+const handlesubmit =(e)=>
+{
+e.preventDefault();
+const list = [todo];
 
-    const todo ="this is the todo....";
+}   
 
   return (
     <div className="container">
-     <h1>the Todo List</h1>
-     <div className="content">
-        <div className="input">
-        <input type="button" value="Add your list..."/>
-        <button className='btn'>ADD</button>
-        </div>    
-        <div className="list">
-            <div className="todo">
-            <p>{todo} </p>
-            </div>
-            <div className="todo">
-            <p>{todo}</p>
-            </div>
-            <div className="todo">
-            <p>{todo}</p>
-            </div>
-            
-        </div>
-        <div className="button">
-          <button className='btn2'>clear all completed tasks</button>
-        </div>
+      
+        <form onSubmit={handlesubmit}>
+          <div className="input">
+            <label>enter your todo: </label>
+            <input onChange={change} 
+            type="text" value={todo
+            }/>
+            <button onClick={clickhandler}>add</button>
+          </div>
+          <select name="" id="">
+            <option value="ronald">ronald</option>
+            <option value="dave"> dave</option>
+          </select>
```

Comment:
This feature is not part of the wireframe I shared with you. Please make sure to implement only and only features that are requested for. For that cse, please consider removing it